api/bin/chainflip-ingress-egress-tracker/src/witnessing/state_chain.rs

```diff
@@ -540,6 +540,7 @@ mod tests {
 						max_priority_fee_per_gas: None,
 						contract: H160::from([0; 20]),
 						gas_limit: None,
+						tx_ref: Default::default(),
```

Comment:
i think the naming is a little bit the other way around in this PR. On these chain specific types, we should use the specific name they refer to. e.g. in this case it should be `tx_hash`, and in polkadot it should be `tx_id`. And then on the generic type i.e. the `BroadcastSuccess` event, the field should be `tx_ref` since it's more general.

Reply:
Yeah sorry about the naming, I wasn't really sure what to use, followed your suggestions, makes everything clearer!